### ISSUE TITLE
Failing DetectorDescriptionRegressionTestDOMCount unit test (needs clean up)

### ISSUE THREAD

**Issue by davidlt** [cms-sw/cmssw#14922]
`DetectorDescriptionRegressionTestDOMCount ` is failing:
```
===== Test "DetectorDescriptionRegressionTestDOMCount" ====
argument 0: DetectorDescriptionRegressionTestDOMCount
argument 1: /bin/bash
argument 2: DetectorDescription/RegressionTest/test
argument 3: run_DOMCount.sh
shell is: /bin/bash
Current directory is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100
topdir is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100
testdir is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/src/DetectorDescription/RegressionTest/test
tmpdir is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/tmp/slc6_amd64_gcc493
testbin is: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/test/slc6_amd64_gcc493
Running script: /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/src/DetectorDescription/RegressionTest/test/run_DOMCount.sh
/tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100/tmp/slc6_amd64_gcc493 /tmp/cmsbuild/workspace/ib-run-qa/CMSSW_8_1_X_2016-06-18-1100
Normal output of DOMCount is written to file tmp/slc6_amd64_gcc493/run_DOMCount.log
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometry2015XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometry2015devXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryAPD1XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryGFlashXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryHFLibraryXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryHFParametrizeXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryNoAPDXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsIdealGeometryTotemT1XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015CastorMeasuredXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015CastorSystMinusXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015CastorSystPlusXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015MuonGEMDevXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015PilotXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015ZeroMaterialXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015devCastorMeasuredXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015devCastorSystMinusXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015devCastorSystPlusXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2015devXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2016XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2017XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2017devXML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2019XML_cfi
run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2023DevXML_cfi
Failure run_DOMCount.py Geometry/CMSCommonData/cmsExtendedGeometry2023DevXML_cfi: status 1
status = 256

---> test DetectorDescriptionRegressionTestDOMCount had ERRORS
```

The file has been removed  almost two weeks ago:
```
commit 53ffef12b0d5e5f3f6db4b828f6734ee80752b9f
Author: Kevin Pedro
Date:   Tue Jun 7 12:17:58 2016 -0500

    remove unsupported 2023 geometries

 .../python/GeometryExtended2023DevReco_cff.py      |   53 ----
 .../Geometry/python/GeometryExtended2023Dev_cff.py |    8 -
 .../python/GeometryExtended2023MuonReco_cff.py     |   35 ---
 .../python/GeometryExtended2023Muon_cff.py         |    9 -
 .../python/GeometryExtended2023Reco_cff.py         |   53 ----
 .../Geometry/python/GeometryExtended2023_cff.py    |   13 -
 .../python/cmsExtendedGeometry2023DevXML_cfi.py    |  291 --------------------
 .../python/cmsExtendedGeometry2023MuonXML_cfi.py   |  290 -------------------
 .../python/cmsExtendedGeometry2023XML_cfi.py       |  288 -------------------
```

```
DetectorDescription/RegressionTest/test/run_DOMCount.sh:cfiFiles="${cfiFiles} Geometry/CMSCommonData/cmsExtendedGeometry2023DevXML_cfi"
```
/cc @kpedro88 @ianna

**Comment by davidlt** [comment]
assign geometry

**Comment by smuzaffar** [comment]
this is fixed here #14906

**Comment by ianna** [comment]
+1
